### PR TITLE
feat: modify autotune, increment workers and fetchers.

### PR DESCRIPTION
**Pull request by ogbanugot**
- Autotune feature to steadily increase workers and fetchers.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,32 +1,49 @@
...
     branches: [v3]
   pull_request:
     branches: [v3]
 
 jobs:
-  build:
-    name: build
-    runs-on: ubuntu-latest
+  test:
+    if: ${{ !(contains(github.head_ref, 'ui/')) || !(contains(github.head_ref, 'cms/')) }}
```
Comment (jirevwe) [inline .github/workflows/build.yml]: Don't think we need this line though

File: .github/workflows/golangci-lint.yml [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-    tags:
-      - v*
-    branches:
-      - master
-      - main
   pull_request:
 
 jobs:
   golangci:
+    if: ${{ !(contains(github.head_ref, 'ui/')) || !(contains(github.head_ref, 'cms/')) }}
```
Comment (jirevwe) [inline .github/workflows/golangci-lint.yml]: This one too

File: sysinfo_other.go [hunk 3]
```diff
@@ -0,0 +1,8 @@
+//go:build !linux
+// +build !linux
+
+package taskq
+
+func hasFreeSystemResources() bool {
```
Comment (jirevwe) [inline sysinfo_other.go:6]: Does this function override the default taskq one?

**Review by jirevwe — CHANGES_REQUESTED** [review]

File: sysinfo_other.go (hunk 3 above)
Comment (ogbanugot) [reply sysinfo_other.go:6]: yes, its the only one being used now actually.

**Review by jirevwe — APPROVED** [review]

**Review by Dotunj — APPROVED** [review]

**Pull request merged**